packages/paste-website/src/component-examples/VisualPickerExamples.ts

```diff
@@ -0,0 +1,845 @@
+                </MediaObject>
+                <Box display="flex" columnGap="space30">
+                  <AvatarGroup size="sizeIcon20" variant="user">
+                    <Avatar name="video recipient1" icon={UserIcon} />
+                    <Avatar name="video recipient2" icon={UserIcon} />
+                    <Avatar name="video recipient3" icon={UserIcon} />
+                  </AvatarGroup>
+                  <Text fontWeight="fontWeightBold">+83</Text>
+                </Box>
+              </Box>
+            </VisualPickerCheckbox>
+            <VisualPickerCheckbox id={useUID()} labelText="email" checked={checked4} onChange={handleChange4}>
+              <Box width="size50" display="flex" justifyContent="space-between" alignItems="center">
+                <MediaObject verticalAlign="center">
+                  <MediaFigure spacing="space50">
+                    <Avatar variant="entity" icon={EmailIcon} size="sizeIcon90" name="code" color="decorative20" />
+                  </MediaFigure>
+                  <MediaBody>
+                    <Text as="div" fontWeight="fontWeightSemibold">
+                      Email
+                    </Text>
+                    <Text as="div" color="colorTextWeak">
+                      6,081 recipients
+                    </Text>
+                  </MediaBody>
+                </MediaObject>
+                <Box display="flex" columnGap="space30">
+                  <AvatarGroup size="sizeIcon20" variant="user">
+                    <Avatar name="email recipient1" icon={UserIcon} />
+                    <Avatar name="email recipient2" icon={UserIcon} />
+                    <Avatar name="email recipient3" icon={UserIcon} />
+                  </AvatarGroup>
+                  <Text fontWeight="fontWeightBold">+6,078</Text>
+                </Box>
+              </Box>
+            </VisualPickerCheckbox>
+        </VisualPickerCheckboxGroup>
+    );
+}
+render(
+    <VisualPickerExample/>
+)
+`.trim();
+
+export const VisualPickerCheckboxHorizontal = `
+const VisualPickerExample = () => {
+    const [checked1, setChecked1] = React.useState(false)
+    const [checked2, setChecked2] = React.useState(false)
+    const [checked3, setChecked3] = React.useState(false)
+    const handleChange1 = (event) => setChecked1(event.target.checked)
+    const handleChange2 = (event) => setChecked2(event.target.checked)
+    const handleChange3 = (event) => setChecked3(event.target.checked)
+    return (
+        <VisualPickerCheckboxGroup
+            orientation="horizontal"
+            name={useUID()}
+            id={useUID()} 
+            legend={
+              <Text as="span" color="currentColor">
+                Select a channel to test
+              </Text>
+            }
+            helpText="The test message will only be sent to test recipients."
+        >
+            <VisualPickerCheckbox id={useUID()} labelText="sms" checked={checked1} onChange={handleChange1}>
```

Comment:
todo: add right padding to each of these items so the content looks more centered.

But also...now that I've looked at a few of these horizontal examples, should items in groups just fill the width of the container equally? Like fitted tabs?

Reply:
Added padding for now but not opposed to making that change in Visual Picker. My only hesitation is that the sizing is completely based on the content right now, like our Card component, and that would still be true for vertical pickers, so it feels a little inconsistent. What do you think?

Reply:
Talked a bit about it with @SiTaggart on Friday—he brought up the point that it's probably unlikely that anyone's using the horizontal group not to have the items all be the same width.

I think the inconsistency is prob fine in this case to make the composition of a horizontal Visual Picker a little less fiddly, especially since the usage of it is less generic than a Card.

Reply:
I think our cards are all full width of the container though, just their height is variable based on content.

I think the both vertical and horizontal should be 100% width of their container.

Reply:
btw let's replace all the horizontal examples with this layout too: https://www.figma.com/file/T8blAjMrVlBc0sPX92T2PQ/(feat)Visual-Picker---add-component?type=design&node-id=32322-163433&mode=design&t=cKanwuuE1ABVLuZ0-4

![image](https://github.com/twilio-labs/paste/assets/3821840/631f9e49-d5c5-4d26-99b3-fcd5aedcb1bf)

The slot area can be replaced with an image, icon, etc.

Reply:
^ Not super obvious here, but it also includes 4px top padding to the checkbox/radio input just for horizontal items.